pkgs/development/python-modules/shippinglabel/default.nix

```diff
@@ -27,13 +29,10 @@ buildPythonPackage rec {
     domdf-python-tools
     packaging
     typing-extensions
+    hatchling
```

Comment:
This is wrong and belongs into build-system https://github.com/domdfcoding/shippinglabel/blob/master/pyproject.toml#L2

Reply:
https://github.com/NixOS/nixpkgs/pull/396303